src/CUDA2HIP_Doc.cpp

```diff
@@ -125,6 +125,17 @@ namespace doc {
   const string sROCSPARSE_csv = sROCSPARSE + csv_ext;
   const string sCUSPARSE = "CUSPARSE";
 
+  /* hipFile / rocFile */
+  const string sHIPFILE             = "cuFile_API_supported_by_HIP";
+  const string sHIPFILE_md          = sHIPFILE + md_ext;
+  const string sHIPFILE_csv         = sHIPFILE + csv_ext;
+  const string sHIPFILE_and_ROC_md  = sHIPFILE + sandROC + md_ext;
+  const string sHIPFILE_and_ROC_csv = sHIPFILE + sandROC + csv_ext;
+  const string sROCFILE             = "cuFile_API_supported_by_ROC";
+  const string sROCFILE_md          = sROCFILE + md_ext;
+  const string sROCFILE_csv         = sROCFILE + csv_ext;
+  const string sCUFILE              = "cuFile";
```

Comment:
```suggestion
  const string sHIPFILE = "cuFile_API_supported_by_HIP";
```

Could we keep same format as before?

Reply:
How is the format different? I think I just copied one of the other libraries that has both hip and roc flavors (perhaps hipSolver/rocSover) because we have both hipFile and rocFile and would like to support both.

Reply:
I guess the request was to keep the existing formatting.

Reply:
> How is the format different? I think I just copied one of the other libraries that has both hip and roc flavors (perhaps hipSolver/rocSover) because we have both hipFile and rocFile and would like to support both.

```suggestion
  const string sHIPFILE = "cuFile_API_supported_by_HIP";
```
I meant to use existing format.

Reply:
Is this all okay now?

Reply:
```suggestion
  const string sHIPFILE = "cuFile_API_supported_by_HIP";
```
still, some part of it needs to change. LGTM with a nit.